Emby.Dlna/Server/DescriptionXmlBuilder.cs

```diff
@@ -189,7 +189,7 @@ private void AppendIconList(StringBuilder builder)
                 builder.Append("<icon>");
 
                 builder.Append("<mimetype>")
-                    .Append(SecurityElement.Escape(icon.MimeType ?? string.Empty))
+                    .Append(SecurityElement.Escape(icon.MimeType))
```

Comment:
This may be null
![image](https://user-images.githubusercontent.com/675118/146776974-e1972ea8-e656-41a2-b306-0cfc526c5869.png)